.travis.yml

```diff
@@ -2,4 +2,4 @@ language: node_js
 node_js:
   - "6"
 before_install:
-  - yarn add react react-dom
+  - yarn add react@^15 react-dom@^15
```

Comment:
Fixing to 15 for now until we update the peer dependency and make sure it's working